doc improvement: General doc fixes (#8641)
nrfconnect/sdk-nrf: samples/bluetooth/central_hr_coded/README.rst

The sample also requires a device running a Heart Rate Server with LE Coded PHY support to connect to.
For example, another development kit running the :ref:`peripheral_hr_coded` sample.

Overview
********

> The sample demonstrates a Bluetooth® LE Central role functionality by scanning for other Bluetooth LE devices that run a Heart Rate Server with LE Coded PHY support, which is not available in Zephyr Bluetooth LE Controller (See :ref:`ug_ble_controller` for more information).
> It then establishes a connection to the first Peripheral device in range.
> It can be used together with the :ref:`peripheral_hr_coded` sample.

Reviewer: ```suggestion
The sample demonstrates a Bluetooth® LE Central role functionality by scanning for other Bluetooth LE devices that run a Heart Rate Server with LE Coded PHY support, which is not available in Zephyr Bluetooth LE Controller.
See :ref:`ug_ble_controller` for more information.
It then establishes a connection to the first peripheral device in range.
You can use it together with the :ref:`peripheral_hr_coded` sample.
```
